index-sample.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sample</title>
+    <link rel="stylesheet" href="/style-sample.css">
+</head>
+<body>
+    <div class="app-sample"> 
+        <img src="/images/imgQuotes.png" alt="img" class="section-quotes">
+        <div class="section-heading">
+            <h2 class="section-title">Real Stories from Real Customers</h2>   
+            <p class="section-desp">Get inspired by these stories.</p>                 
+        </div>        
+        <div class="section-testimonial3">
+            <img src="/images/imgHubspot.png" alt="img" class="testimonial-image">
+            <div class="section-content">
+                <img src="/images/imgQuotes.png" alt="img" class="section-quotes-testimonial">
+                <div class="section-feedbackPerson">
+                    <p class="section-feedback">To quickly start my startup landing page design, I was looking for a landing page UI Kit. Landify is one of the best landing page UI kit I have come across. It’s so flexible, well organised and easily editable.</p>
+                    <img src="/images/imgPersonDetails.png" alt="img" class="section-persondetails">
+                </div>
+            </div>
+        </div>
+        <div class="section-testimonial2">
+            <img src="/images/imgBookmyshow.png" alt="img" class="testimonial-image">
+            <div class="section-content">
+                <img src="/images/imgQuotes.png" alt="img" class="section-quotes-testimonial">
+                <div class="section-feedbackPerson">
+                    <p class="section-feedback">Landify saved our time in designing my company page.</p>
+                    <img src="/images/imgTest2.png" alt="img" class="section-persondetails">
```

Comment:
This one too. This is a text so use appropriate text here instead of an image that doesn't fit well.